### ISSUE TITLE
Add ASET PRS props to ASET-Consolidated-Props

### ISSUE THREAD

**Issue by kurgut** [StoneBlue/ASET-Consolidated-Props#26]
I noticed ASET ERS comes with 2 very nice props, the `ERSDriveMonitorScreen` and `ERSPitchRollDisplay`.
Would it be possible, if modders (like me right now ^^) would like to include those props in their IVA, to include them to Aset-Props ?
To avoid adding ERS as a dependency for the user.
So maybe it would require to take those props out of ERS, so they're not duplicated.
I guess I'll issue on ERS too and link it to here..
Thank you !

**Comment by linuxgurugamer** [comment]
The ERS license is an CC-SA-NC, so I would say, go ahead.  I would suggest making sure there aren't any name collisions, and be sure to reference the contributions from ERS

**Comment by linuxgurugamer** [comment]
Fine by me, but I'm not planning on making any changes to  ERS.  Sounds like these changes are all on the Props side

**Comment by kurgut** [comment]
> Can you share more about your plans @kurgut ?
I was planning implementing them in the External Command Seat IVA mod I started working on, which would include a rover variant, with those props.
Also on a high tech flavor version of PCR.

>Maybe a better path for you would be to create new versions of these props, using the ERS ones as a starting point, and just give attribution?
I'm fine with that too

**Comment by JonnyOThan** [comment]
Yeah, I think just building new rover hud props that are designed to be generalized for all rovers is the right move, and I'd be happy to include them in ASET if you want. Sound good?
